scripts/setup3PoolAdminFee.ts

```diff
@@ -0,0 +1,70 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+// When running the script with `hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+import {
+  ethers,
+  // , run
+} from "hardhat";
+
+async function main(): Promise<void> {
+  // Hardhat always runs the compile task when running scripts through it.
+  // If this runs in a standalone fashion you may want to call compile manually
+  // to make sure everything is compiled
+  // await run("compile");
+  // We get the contract to deploy
+
+  // Constants
+  const [_, deployer] = await ethers.getSigners();
+  console.log(`Deploying contracts with ${deployer.address}`);
+
+  const AmpUtilsFactory = await ethers.getContractFactory("AmplificationUtils", deployer);
+  const amplificationUtils = await AmpUtilsFactory.attach("0x114ECaa70256aFAd393f733aA4B4bF61c8959fc2");
+  console.log(`amplificationUtils attached at ${amplificationUtils.address}`);
+
+  const SwapUtilsFactory = await ethers.getContractFactory("SwapUtils", deployer);
+  const swapUtils = await SwapUtilsFactory.attach("0x0564d68404608599e8c567A0bD74F90a942A69A0");
+  console.log(`swapUtils attached at ${swapUtils.address}`);
+
+  const SwapFlashLoanFactory = await ethers.getContractFactory("SwapFlashLoan", {
+    libraries: {
+      SwapUtils: swapUtils.address,
+      AmplificationUtils: amplificationUtils.address,
+    },
+  });
+
+  const swapFlashLoan = await SwapFlashLoanFactory.connect(deployer).attach(
+    "0x458459E48dbAC0C8Ca83F8D0b7b29FEfE60c3970",
+  );
+  console.log(`swapFlashLoan connected and attached at "${swapFlashLoan.address}"`);
+
+  // Set Fee Address to deployer
+  await swapFlashLoan.setFeeAddress(deployer.address);
```

Comment:
no need to do this: https://github.com/trisolaris-labs/trisolaris_core/blob/main/contracts/stableswap/Swap.sol#L189

we do this anywats when we initialize the contract